### DISCUSSION TITLE
A lot of pending messages

### DISCUSSION THREAD

**Discussion by creative-andrew** [postalserver/postal#1552] [Help with using Postal] [answered]
Hi everyone,

I am getting a lot of pending messages:

![image](https://user-images.githubusercontent.com/18492002/130033578-e8161d9f-bbd9-46c6-9910-4cbaf4928254.png)

Any idea what could be causing this?

**Comment by willpower232** [comment, marked as answer]
do you know how many workers you have running currently?

**Reply by creative-andrew** [reply]
![image](https://user-images.githubusercontent.com/18492002/130037187-8352d638-6c04-4c34-840f-c00f80b147e1.png)

This is what I see when I run postal status

@willpower232

**Reply by adamcooke** [reply]
Check the logs for your requeuer process - it clearly not very happy there.

**Reply by yukzanaly** [reply]
I also the same problem. the postal_requeuer_1 keeps showing state Exit 1

I tried to postal logs: `postal logs requeuer` it shows this error `Bunny::AuthenticationFailureError: Authentication with RabbitMQ failed. Please check your connection settings`

how can I check and verify RabbitMQ authentication ?

**Reply by willpower232** [reply]
@yukzanaly have you entered the right details in the config file?

**Reply by yukzanaly** [reply]
@willpower232 It turned out the RabbitMQ user and password were wrong. I don't know why. I was previously working until it wasn't.

So I created a new rabbitmq user and password. then I edit the config file with the new rabitmq user and password. 

Now it works like charm. Thanks
